clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -106,6 +108,20 @@ class IoUtil(storage: Storage) {
   def googleObjectExists(path: URI): Boolean = {
     getBlob(path).exists(_.exists())
   }
+
+  def listGoogleObjects(path: URI): Seq[URI] = {
+    val blobId = IoUtil.toBlobId(path)
```

Comment:
This should probably be `requireBlo` instead of `toBlobId`. `toBlobId` might return `null`.